docs/scripts/readme_plot.py

```diff
@@ -28,7 +30,7 @@ def __init__(self, unique_id, model):
     def step(self):
         # Verify agent has some wealth
         if self.wealth > 0:
-            other_agent = self.random.choice(self.model.schedule.agents)
+            other_agent = choice(self.model.agents)
```

Comment:
Mesa's default random generator is the stdlib `random`, so it's fine to use `self.random.choice`. And yes, I'm aware that for small sample size, the stdlib is faster than NumPy.